scripts/tests/flow/flow_tests_whitelist.txt

```diff
@@ -12,6 +12,7 @@ ES6/binding-pattern/object-pattern/await-prop-in-async-function.js
 JSX_invalid/migrated_0000.js
 arrow_function_invalid/migrated_0002.js
 async_await/async_generic_method.js
+async_await/migrated_0007.js
```

Comment:
I think that this test is wrong (because of https://tc39.github.io/ecma262/#sec-identifiers-static-semantics-early-errors)
https://github.com/facebook/flow/blob/master/src/parser/test/flow/async_await/migrated_0007.js